Dockerfile

```diff
@@ -46,4 +46,7 @@ RUN apt-get remove -y \
     cpp-12 \
     cpp
 
+# Disable analytics in OSS
+RUN export GX_ANALYTICS_ENABLED=false
```

Comment:
Any reason to not use the `ENV` instruction? https://docs.docker.com/reference/dockerfile/#env

Reply:
Oh good catch, you should use ENV. I think RUN just sets it for this session when building the image but ENV will set it in the resultant image. At least according to this: https://stackoverflow.com/questions/33379393/docker-env-vs-run-export